src/pages/search/ListingCard.tsx

```diff
@@ -5,24 +5,27 @@ import { ListingShort } from 'networking/listings';
 
 const ListingCard = ({
   homeType,
+  idSlug,
   listingPicUrl,
   pricePerNightUsd,
   title
 }: ListingShort) => (
-  <Card tag={Fade} className="w-100">
-    <div className="embed-responsive embed-responsive-4by3">
-      <div className="embed-responsive-item">
-        <CardImg className="w-100" src={listingPicUrl} alt={`Photo of ${title}`} />
+  <a href={`/work/listings/${idSlug}`} className="w-100 h-100">
```

Comment:
Should this be a `BeeLink to` (are we keeping `BeeLink` into the new design?)

Reply:
@kcvan
I think we are using `Link` from 'react-router-dom'

Reply:
I've been using `<Link to="/blah">` for internal routing, and ill add `<Link tag="a" href="https://vicDaBes.com">` for any external routing

Reply:
Thanks, changed!